var/spack/repos/builtin/packages/r-tictoc/package.py

```diff
@@ -20,9 +20,9 @@ class RTictoc(RPackage):
     'push', 'pop', 'first', 'last' and 'clear'."""
 
     homepage = "https://collectivemedia.github.io/tictoc/"
-    url      = "https://cloud.r-project.org/src/contrib/tictoc_1.0.tar.gz"
-    list_url = "https://cloud.r-project.org/src/contrib/Archive/tictoc"
+    cran = "tictoc"
 
+    version('1.0.1', sha256='a09a1535c417ddf6637bbbda5fca6edab6c7f7b252a64e57e99d4d0748712705')
     version('1.0', sha256='47da097c1822caa2d8e262381987cfa556ad901131eb96109752742526b2e2fe')
 
     depends_on('r@3.0.3:', type=('build', 'run'))
```

Comment:
This builds with `r@:4.0`.
```suggestion
    depends_on('r@3.0.3:, type=('build', 'run'), when='@1.0.1:')
    depends_on('r@3.0.3:4.0', type=('build', 'run'), when='@1.0')
```